service/templates/cloudformation/worker_policy.yaml

```diff
@@ -9,12 +10,12 @@
           Principal:
             Service: "ec2.amazonaws.com"
           Action: "sts:AssumeRole"
-  workerRolePolicy:
+  WorkerRolePolicy:
```

Comment:
I like that this makes the resource names consistent and starting with a capital. Should we do the same for the R53 recordsets?

Reply:
Makes perfect sense, fixed